samples/basic/rgb_gradient/sample.yaml

```diff
@@ -0,0 +1,10 @@
+sample:
+  name: RGB LED GRADIENT
+tests:
+  sample.basic.rgb_gradient:
+    filter: dt_alias_exists("red-pwm-led") and
+            dt_alias_exists("green-pwm-led") and
+            dt_alias_exists("blue-pwm-led")
+    tags: drivers pwm
+    depends_on: pwm
+    harness: led
```

Comment:
No harness needed?

Reply:
the other samples using LEDs have this harness, including rgb_led, blinky, blinky_pwm, and fade_led